CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+x.x.x Release notes (yyyy-MM-dd)
+=============================================================
+
+### Breaking Changes
+
+* None.
+
+### Enhancements
+
+* Expose `RLMObject.object(forPrimaryKey:)` as a factory method for Swift so
+  that it is callable with recent versions of Swift.
+
+### Bugfixes
+
+* Exclude the RLMObject-derived Permissions classes from the types repored by
```

Comment:
I think this entry is here by mistake?

Reply:
That was added by #5691 which was missing a changelog entry.